src/Connection.php

```diff
@@ -114,13 +114,31 @@ public function getSchemaBuilder()
     /**
      * Get the MongoDB database object.
      *
+     * @deprecated Use getDatabase() instead
```

Comment:
This is the beginning of [PHPORM-278](https://jira.mongodb.org/browse/PHPORM-278), but I should create a distinct PR.

Reply:
https://github.com/mongodb/laravel-mongodb/pull/3289